core/src/main/java/nl/pim16aap2/bigDoors/moveBlocks/SlidingMover.java

```diff
@@ -219,7 +222,7 @@ public void run()
                     for (MyBlockData savedBlock : savedBlocks)
                         if (!savedBlock.getMat().equals(Material.AIR))
                             savedBlock.getFBlock().setVelocity(new Vector(0D, 0D, 0D));
-                    Bukkit.getScheduler().callSyncMethod(plugin, () ->
+                    BigDoors.getScheduler().callSyncMethod(() ->
```

Comment:
Couldn't this use `TaskSchedule#runTask(Location, Runnable)` with the engine as the location, as we already know the area in which it'll happen?

I guess this also goes for the others.

Reply:
changed in latest commit